### ISSUE TITLE
The train_ember.py file is not installed

### ISSUE THREAD

**Issue by ezequielal39** [endgameinc/ember#95]
Hello I am using python 3.6.8
I follow the installation procedure

pip install -r requirements.txt
python setup.py install

The train_ember.py file is not installed
I can't continue the process because I can't train ember_dataset_2018_2.tar.bz2 because it is missing
train_ember.py

Does anyone know why it won't install?

The data that is installed:

C:\enber>python setup.py install
running install
running bdist_egg
running egg_info
writing ember.egg-info\PKG-INFO
writing dependency_links to ember.egg-info\dependency_links.txt
writing top-level names to ember.egg-info\top_level.txt
reading manifest file 'ember.egg-info\SOURCES.txt'
writing manifest file 'ember.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
running build_py
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\ember
copying build\lib\ember\features.py -> build\bdist.win-amd64\egg\ember
copying build\lib\ember\__init__.py -> build\bdist.win-amd64\egg\ember
byte-compiling build\bdist.win-amd64\egg\ember\features.py to features.cpython-36.pyc
byte-compiling build\bdist.win-amd64\egg\ember\__init__.py to __init__.cpython-36.pyc
creating build\bdist.win-amd64\egg\EGG-INFO
copying ember.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying ember.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying ember.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying ember.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist\ember-0.1.0-py3.6.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing ember-0.1.0-py3.6.egg
Removing c:\users\mabel\appdata\local\programs\python\python36\lib\site-packages\ember-0.1.0-py3.6.egg
Copying ember-0.1.0-py3.6.egg to c:\users\mabel\appdata\local\programs\python\python36\lib\site-packages
ember 0.1.0 is already the active version in easy-install.pth

03/01/2023  08:14    <DIR>          .
03/01/2023  08:14    <DIR>          ..
03/01/2023  08:14    <DIR>          build
03/01/2023  08:14    <DIR>          dist
03/01/2023  08:04               442 Dockerfile
03/01/2023  08:04    <DIR>          ember
03/01/2023  17:23    <DIR>          ember.egg-info
03/01/2023  08:04               226 LICENSE.txt
03/01/2023  08:04    <DIR>          licenses
03/01/2023  08:04    <DIR>          malconv
03/01/2023  08:04             7.534 README.md
03/01/2023  08:04                91 requirements.txt
03/01/2023  08:04                94 requirements_conda.txt
03/01/2023  08:04                78 requirements_notebook.txt
03/01/2023  08:04    <DIR>          resources
03/01/2023  08:04    <DIR>          scripts
03/01/2023  08:04               503 setup.py
               7 archivos          8.968 bytes

**Comment by ezequielal39** [comment]
Lo instale con el comando conda y funciona bien

**Comment by Salma1795** [comment]
I have the same problem... Kindly inform me if you find the solution

**Comment by srao-cipher** [comment]
Yup - not seeing it in the source code anywhere. Not sure if it was just renamed to init_ember.py

**Comment by anirudh1261** [comment]
Yeah I was Also facing the Same Problem
